projects/storefrontlib/src/shared/components/form-components/item-counter/item-counter.component.ts

```diff
@@ -101,7 +105,6 @@ export class ItemCounterComponent
    * Function set 'isValueOutOfRange' flag and adjust value in range. Then update model value and refresh input
    */
   manualChange(newValue: number): void {
-    this.isValueOutOfRange = this.isOutOfRange(newValue);
```

Comment:
There is one more mention of this flag. Could you remove it from the comment above `manualChange` function?